stdlib/codecs.pyi

```diff
@@ -253,15 +251,15 @@ class StreamRecoder(BinaryIO):
     def readlines(self, sizehint: int | None = ...) -> list[bytes]: ...
     def __next__(self) -> bytes: ...
     def __iter__(self: Self) -> Self: ...
-    def write(self, data: bytes) -> int: ...
-    def writelines(self, list: Iterable[bytes]) -> int: ...  # type: ignore  # it's supposed to return None
+    def write(self, data: bytes) -> None: ...  # type: ignore[override]
+    def writelines(self, list: Iterable[bytes]) -> None: ...
     def reset(self) -> None: ...
     def __getattr__(self, name: str) -> Any: ...
     def __enter__(self: Self) -> Self: ...
     def __exit__(self, type: type[BaseException] | None, value: BaseException | None, tb: types.TracebackType | None) -> None: ...
+    def seek(self, offset: int, whence: int = ...) -> None: ...  # type: ignore[override]
```

Comment:
`seek` does actually exist directly, which is why I've moved it above the comment.